linuxkm/linuxkm_wc_port.h

```diff
@@ -146,6 +263,18 @@
     #include <linux/net.h>
     #include <linux/slab.h>
 
+    #ifdef LINUXKM_LKCAPI_REGISTER
+        #include <linux/crypto.h>
+        #include <linux/scatterlist.h>
+        #include <crypto/scatterwalk.h>
+        #include <crypto/internal/aead.h>
+        #include <crypto/internal/skcipher.h>
+
+        #ifndef WC_AES_XTS_SUPPORT_SIMULTANEOUS_ENC_AND_DEC_KEYS
```

Comment:
Please add inline documentation here for this macro.

Reply:
:+1:

Reply:
fixed.